templates/repo/issue/view_content/comments.tmpl

```diff
@@ -364,8 +364,8 @@
 				</span>
 				{{if .DependentIssue}}
 					<div class="detail">
-						<span class="text grey muted-links">{{svg "octicon-trash"}}</span>
-						<span class="text grey muted-links">
+						<span class="text grey muted-links gt-df">{{svg "octicon-trash"}}</span>
+						<span class="text grey muted-links gt-df">
```

Comment:
It doesn't seem right by doing so. There should be no difference.

Reply:
Here it's essentially a noop, yes. I guess it was done for consistency.